posthog/migrations/0435_annotation_dashboard_alter_annotation_scope.py

```diff
@@ -0,0 +1,93 @@
+# Generated by Django 4.2.11 on 2024-07-10 12:07
```

Comment:
Have rewritten this according to [here](https://github.com/PostHog/posthog/blob/master/posthog/migrations/0415_pluginconfig_match_action.py) to remove the `CONSTRAINT` command